object/check.go

```diff
@@ -256,6 +268,10 @@ func CheckLdapUserPassword(user *User, password string, lang string) (string, er
 	userDisabled := false
 
 	for _, ldapServer := range ldaps {
+		if ldapId != "" && ldapServer.Id != ldapId {
+			continue
+		}
```

Comment:
if we have ldapId -> we can query only 1 ldapServer by id from db.

Reply:
done